examples/bri-3/src/bri/identity/bpiSubjects/agents/bpiSubjects.agent.ts

```diff
@@ -1,22 +1,36 @@
-import { BadRequestException, Injectable } from "@nestjs/common";
-import { BpiSubject } from "../models/bpiSubject";
-import { BpiSubjectType } from "../models/bpiSubjectType.enum";
+import { BadRequestException, Injectable } from '@nestjs/common';
+import { BpiSubject } from '../models/bpiSubject';
+import { BpiSubjectType } from '../models/bpiSubjectType.enum';
+
+import { uuid } from 'uuidv4';
 
-import { uuid } from 'uuidv4'; 
- 
 @Injectable()
 export class BpiSubjectAgent {
-  // Agent methods have extremely declarative names and perform a single task 
-  public throwIfCreateBpiSubjectInputInvalid(name: string, desc: string, pk: string) {
+  // Agent methods have extremely declarative names and perform a single task
+  public throwIfCreateBpiSubjectInputInvalid(
+    name: string,
+    desc: string,
```

Comment:
I think it would be a small refactoring task to rename desc -> description, and pk -> publicKey. Can we push in this change?